tools/src/test/java/org/apache/kafka/tools/consumer/group/ResetConsumerGroupOffsetTest.java

```diff
@@ -136,10 +136,6 @@ public void testResetOffsetsNotExistingGroup(ClusterInstance cluster) throws Exc
         String[] args = buildArgsForGroup(cluster, group, "--all-topics", "--to-current", "--execute");
 
         try (ConsumerGroupCommand.ConsumerGroupService service = getConsumerGroupService(args)) {
-            // Make sure we got a coordinator
-            TestUtils.waitForCondition(
-                    () -> "localhost".equals(service.collectGroupState(group).coordinator.host()),
-                    "Can't find a coordinator");
```

Comment:
That's because `collectGroupState` fails for a non-existent group now.  The `resetOffsets` method handles a non-existent group directly.

I'm going to do a little more work in this area since I'm not totally happy with the output from `kafka-consumer-groups.sh` when the group ID is a share group. It works but it's a bit scruffy.